### PR TITLE
PORTALS-3176

### PR DESCRIPTION
**Pull request by jay-hodgson**

### CODE REVIEW

File: packages/synapse-react-client/src/components/DatasetJsonLdScript.tsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    version = versionFromProps
+  } else if (searchParams) {
+    entityId = searchParams['id']
+    if (searchParams['version']) {
+      version = parseInt(searchParams['version'])
+    }
+  }
+  const { data: entityBundle } = useGetEntityBundle(entityId!, version)
+  const { data: wikiPage } = useWikiPage(entityId!)
+  const html = md.renderInline(wikiPage?.markdown ?? '')
```
Comment (nickgros) [inline packages/synapse-react-client/src/components/DatasetJsonLdScript.tsx:54]: I could see us re-using and gradually improving a utility to get plain text from Synapse wiki markdown, so you may consider extracting some of this logic into its own file

File: packages/synapse-react-client/src/synapse-queries/wiki/useWikiPage.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { ObjectType, WikiPage } from '@sage-bionetworks/synapse-types'
+import { useQuery, UseQueryOptions } from '@tanstack/react-query'
+import { SynapseClientError } from '../../utils/SynapseClientError'
+import { useSynapseContext } from '../../utils'
+import SynapseClient from '../../synapse-client'
+
+export function useWikiPage(
```
Comment (nickgros) [inline packages/synapse-react-client/src/synapse-queries/wiki/useWikiPage.ts:7]: Is this different from useWiki.ts `useGetWikiPage`?

File: packages/synapse-react-client/src/utils/hooks/useJsonLdScriptElement.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import { useEffect } from 'react'
+import { Thing, WithContext } from 'schema-dts'
```
Comment (nickgros) [inline packages/synapse-react-client/src/utils/hooks/useJsonLdScriptElement.ts:2]: These JSON-LD types seem useful to have

**Review by nickgros — APPROVED** [review]

File: packages/synapse-react-client/src/synapse-queries/wiki/useWikiPage.ts (hunk 2 above)
Comment (jay-hodgson) [reply packages/synapse-react-client/src/synapse-queries/wiki/useWikiPage.ts]: Nice catch.  I pulled in changes from an old PR, perhaps useGetWikiPage was unavailable at that time.  Replaced with useWiki.useGetWikiPage()!

**Pull request merged**